data/assets/scene/digitaluniverse/sdss.asset

```diff
@@ -43,7 +43,7 @@ local Object = {
     }
   },
   GUI = {
-    Name = "Sloan Digital Sky Survey",
+    Name = "Sloan Galaxies",
```

Comment:
```suggestion
    Name = "Sloan Digital Sky Survey",
```

It is the name of the survey

Reply:
The problem here is Sloan DSS is much larger than just galaxies. I used to have a data set of Sloan stars long ago. So, I want to keep our options open that way. Also, these asset names take on the quality of a name... they need not be absolutely precise, but must have a name that is simple and meaningful. I think Sloan Galaxies is unmistakably from the SDSS.

Reply:
That is *very* verbose. Also, until a short time ago, the quasar catalog was from Sloan too. So, we always had "Sloan Galaxies" and Sloan Quasars". How about "Sloan DSS Galaxies"? Though I feel the "DSS" is a bit redundant and perhaps cryptic to those unfamiliar with the acronym SDSS.

In the end, the actual name doesn't much matter. We just want to identify that these are the galaxies that the Sloan Foundation funded, and we want something that rolls off the tongue, and is concise. Each of these names has to fit on one of those "cards" in the documentation, and a lengthy name just doesn't look good.

Reply:
I think Sloan DSS Galaxies can work for me :+1:

Reply:
OK. I'll make those changes.